public/app/features/search/components/DashboardActions.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
+import { HorizontalGroup } from '@grafana/ui';
+
+export interface Props {
+  folderId?: number;
+  isEditor: boolean;
+  canEdit: boolean;
+}
+
+export const DashboardActions: FC<Props> = ({ folderId, isEditor, canEdit }) => {
+  const actionUrl = (type: string) => {
+    let url = `dashboard/${type}`;
+
+    if (folderId) {
+      url += `?folderId=${folderId}`;
+    }
+
+    return url;
+  };
+
+  return (
+    <HorizontalGroup spacing="md" align="center">
+      {canEdit && (
+        <a className="btn btn-primary" href={actionUrl('new')}>
```

Comment:
Use `LinkButton` here :)

Reply:
Ah, nice! Let me know if there's any other form elements I need to update :)